addons/web/static/src/core/model_selector/model_selector.scss

```diff
@@ -0,0 +1,10 @@
+.o_sp_input_dropdown {
+    .o-autocomplete--dropdown-menu {
+        width: 25ch;
+        max-height: 27% !important;
```

Comment:
this looks suspiciously precise. Isn't the style already properly managed by the Autocomplere component ?